_data/tabs.yml

```diff
@@ -2,19 +2,21 @@
   link: /
 
 - name: About
-  link: /about/
+  dropdown:
```

Comment:
BTW, I intentionally didn't put extra newlines here b/c when I added them it was hard to see that they were part of the dropdown visually; this makes it clearly look like they're related

Reply:
Makes sense to me. :+1: